api/provider/aws/task_delete.go

```diff
@@ -1,5 +1,49 @@
 package aws
 
-func (t *TaskProvider) Delete(TaskID string) error {
+import (
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/ecs"
+)
+
+func (t *TaskProvider) Delete(taskID string) error {
+	environmentID, err := t.lookupTaskEnvironmentID(taskID)
+	if err != nil {
+		return err
+	}
+	fqEnvironmentID := addLayer0Prefix(t.Config.Instance(), environmentID)
+
+	taskARN, err := t.lookupTaskARN(taskID)
+	if err != nil {
+		return err
+	}
+
+	clusterName := fqEnvironmentID
```

Comment:
[super nit] To group similar concepts/work/actions/objects together, we can move this line up to be just under `fqEnvironmentID := ...`.

Reply:
Actually, ignore this comment - our RoT sets up final arguments for helper functions just above the function calls; this looked strange to me because there's only one var being set up here.